docs/deploy.rst

```diff
@@ -0,0 +1,229 @@
+
+Look for available products
+===========================
+
+Now that you've set a region, it's time to look for available products in that
+region.
+
+Run the ``croud products`` command and pass in ``region`` to look for available
+products:
+
+.. code-block:: console
+
+   sh$ croud products list --region bregenz.a1
+   +-------------------------------------------------------------+----------+-------------------+---------------------------+--------+
+   | description                                                 | kind     | name              | scale_summary             | tier   |
+   |-------------------------------------------------------------+----------+-------------------+---------------------------+--------|
+   | A CrateDB cluster suited for general purpose workloads      | cluster  | cratedb.az-gp1    | 0 - 0 (1 - 1 nodes)       | xxs    |
+   | A CrateDB cluster suited for general purpose workloads      | cluster  | cratedb.az-gp1    | 0 - 1 (1 - 2 nodes)       | xs     |
+   | A CrateDB cluster suited for general purpose workloads      | cluster  | cratedb.az-gp1    | 0 - 1 (3 - 5 nodes)       | s      |
+   | A CrateDB cluster suited for general purpose workloads      | cluster  | cratedb.az-gp1    | 0 - 2 (3 - 7 nodes)       | m      |
+   | A CrateDB cluster suited for general purpose workloads      | cluster  | cratedb.az-gp1    | 0 - 6 (3 - 15 nodes)      | l      |
+   | A CrateDB cluster suited for general purpose workloads      | cluster  | cratedb.az-gp1    | 0 - 10 (3 - 23 nodes)     | xl     |
+   | A CrateDB cluster with standard storage                     | cluster  | cratedb.az1       | 0 - 1 (1 - 3 nodes)       | xs     |
+   | A CrateDB cluster with standard storage                     | cluster  | cratedb.az1       | 0 - 1 (3 - 5 nodes)       | s      |
+   | A CrateDB cluster with standard storage                     | cluster  | cratedb.az1       | 0 - 2 (3 - 7 nodes)       | m      |
+   | A CrateDB cluster with standard storage                     | cluster  | cratedb.az1       | 0 - 3 (3 - 9 nodes)       | l      |
+   | Crate.io's Offering to connect Azure's EventHub to CrateDB. | consumer | eventhub-consumer | 1 - 1 (1 - 1 instances)   | xs     |
+   | Crate.io's Offering to connect Azure's EventHub to CrateDB. | consumer | eventhub-consumer | 1 - 3 (1 - 3 instances)   | s      |
+   | Crate.io's Offering to connect Azure's EventHub to CrateDB. | consumer | eventhub-consumer | 1 - 6 (1 - 6 instances)   | m      |
+   | Crate.io's Offering to connect Azure's EventHub to CrateDB. | consumer | eventhub-consumer | 1 - 18 (1 - 18 instances) | l      |
+   +-------------------------------------------------------------+----------+-------------------+---------------------------+--------+
+
+The output lists all available CrateDB products in your specified region. All
+products ship in four different Node sizes (node tiers).
+
+Pick the smallest tier which is the ``xxs`` tier. In this case, you would be
```

Comment:
the smallest tier is not xxs. This xxs tier is not available in production (only in dev)